conda_forge_tick/pypi_name_mapping.py

```diff
@@ -0,0 +1,110 @@
+"""
+Builds and maintains mapping of pypi-names to conda-forge names
+
+1: Pacakges should be build from a `https://pypi.io/packages/` source
+2: Packages MUST have a test: imports section importing it
+"""
+
+import glob
+import json
+import sys
+import yaml
+import pathlib
+
+from typing import Dict, List
+
+
+def load_node_meta_yaml(filename: str) -> List[Dict[str, str]]:
+    node_attr = json.load(open(filename))
```

Comment:
you need to use the in-house load here, we have special types (namely set)

Reply:
we sorta do, but they're not used for these :D

Reply:
Perhaps, but if we change the underlying json bindings it might be safer to use the in house loaders.